src/navigation/navigation.es6

```diff
@@ -159,7 +159,9 @@ const initLoginButton = (root) => {
 
       state.currency = data.authorize.currency;
       local_storage.set('currency', state.currency);
-      update_balance(data);
+      const loginId = local_storage.get('authorize').loginid;
+      if (data.authorize && data.authorize.loginid === loginId) update_balance(data);
```

Comment:
```suggestion
      if (data.authorize?.loginid === loginId) update_balance(data);
```

Reply:
@mehrzad-fs  jan as I explained above, the webtrader does not support the optional chaining, so we couldn't use what mentioned unfortunately :(